Add VOLTOCONFIG Env Var (#5752)
plone/volto: docs/source/configuration/volto-config-js.md

  theme
};
```

> ## Using `VOLTOCONFIG` environment variable

Reviewer: ```suggestion
## `VOLTOCONFIG` environment variable usage
```

> You can also provide a custom location for `volto.config.js` via an environment variable.

Reviewer: ```suggestion
This environment variable allows you to specify a custom location for {file}`volto.config.js`.
```

It can be relative to the current project or absolute.

```shell
VOLTOCONFIG=../../volto.config.js yarn start
```

```shell
VOLTOCONFIG=$(pwd)/volto.config.js yarn start
```

> you can also do it in core (monorepo):

Reviewer: I think you mean this, but maybe not. I'm not really sure of your intent.

```suggestion
You can also set it from the root of the monorepo:
```
